### DISCUSSION TITLE
installing yarn2 (not 3)

### DISCUSSION THREAD

**Discussion by jeffrson** [yarnpkg/berry#3280] [Q&A] [answered]
I'm using yarn 1 (1.2.11) to bootstrap yarn 2.

Unfortunately `yarn set version 2` installs berry, which currently is 3.0.0. Now I can use `yarn set version 2` another time, in order to receive current 2.4.2.

Is this really intended?

**Comment by Eomm** [comment]
I'm having the same behaviour.

I spot this comment https://github.com/yarnpkg/berry/issues/2935#issuecomment-880580057

> yarn set version berry && yarn set version 2.4.2

**Comment by jeffrson** [comment, marked as answer]
Yes, this is what I found as a workaround - actually rather ugly...

```
yarn set version 2 && yarn set version 2
```

...using "berry" is only slightly better

**Reply by merceyz** [reply]
This is the correct solution, support for specific v2 versions was added in v2 and we don't plan on backporting it to v1. Support for ranges was added in v3 so if you want to install the latest v2 version your snippet is the correct one

**Comment by jeffrson** [comment]
Meanwhile I found this
https://github.com/yarnpkg/berry/issues/2741

Well, it does'nt seem to be quite bad to use yarn3 anyway - if it was'nt for the change of pnp.js to pnp.cjs.
